wavelength-plugins/profile.js

```diff
@@ -480,56 +522,83 @@ exports.commands = {
 		function getFlag(userid) {
 			let ip = (Users(userid) ? geoip.lookup(Users(userid).latestIp) : false);
 			if (!ip || ip === null) return '';
-			return '<img src="http://flags.fmcdn.net/data/flags/normal/' + ip.country.toLowerCase() + '.png" alt="' + ip.country + '" title="' + ip.country + '" width="20" height="10">';
+			return `<img src="http://flags.fmcdn.net/data/flags/normal/${ip.country.toLowerCase()}.png" alt="${ip.country}" title="${ip.country}" width="20" height="10">`;
 		}
 
 		function background(user) {
 			let bg = Db.backgrounds.get(user);
-			if (!Db.backgrounds.has(user)) return '<div>';
-			return '<div style="background:url(' + bg + ')">';
+			if (!Db.backgrounds.has(user)) return `<div>`;
+			return `<div style="background:url(${bg}); size: 100%">`;
+		}
+
+		function pColor(user) {
+			let color = Db.profilecolor.get(user);
+			if (!Db.profilecolor.has(user)) return '';
+			return `<font color="${color}">`;
 		}
 
 		function song(user) {
 			if (!Db.music.has(user)) return '';
 			let song = Db.music.get(user)['link'];
 			let title = Db.music.get(user)['title'];
-			return '<acronym title="' + title + '"><br /><audio src="' + song + '" controls="" style="width:100%;"></audio></acronym>';
+			return `<acronym title="${title}"><br /><audio src="${song}" controls="" style="width:100%;"></audio></acronym>`;
 		}
 
 		function showProfile() {
 			Economy.readMoney(toId(username), currency => {
-				let profile = '';
-				profile += background(toId(username)) + showBadges(toId(username));
-				profile += '<div style="display: inline-block; width: 7em; vertical-align: 2.5em;"><img src="' + avatar + '" height="80" width="80" align="left"></div>';
-				profile += '<div style="display: inline-block; float: Center">&nbsp;<font color="#24678d"><b>Name:</b></font> ' + WL.nameColor(username, true) + '&nbsp;' + getFlag(toId(username)) + ' ' + showTitle(username) + '<br />';
-				profile += '&nbsp;<font color="#24678d"><b>Group:</b></font> ' + userGroup + ' ' + devCheck(username) + vipCheck(username) + '<br />';
-				profile += '&nbsp;<font color="#24678d"><b>Registered:</b></font> ' + regdate + '<br />';
-				profile += '&nbsp;<font color="#24678d"><b>' + currencyPlural + ':</b></font> ' + currency + '<br />';
+				let profile = ``;
+				profile += `${background(toId(username))} ${showBadges(toId(username))}`;
+				profile += `<div style="display: inline-block; width: 7em; vertical-align: 2.5em;"><img src="${avatar}" height="80" width="80" align="left"></div>`;
+				profile += `&nbsp;${pColor(toId(username))}<b>Name:</b></font> ${WL.nameColor(username, true)}&nbsp; ${getFlag(toId(username))} ${showTitle(username)}<br />`;
```

Comment:
Instead of spamming pColor, why not just use it once since it changes the text color of all messages right? correct me if i'm wrong.
Also you have unused `</font>` tags if pColor returns `''`

Reply:
I'll fix this

Reply:
This line in particular only needs the title ``Name:`` to be a color as the username, title, dev status, and VIP status all have unique colors